tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -418,12 +424,11 @@ generate_qemu_options() {
 	# Support Ceph RADOS Block Device (RBD)
 	[ -z "${static}" ] && qemu_options+=(functionality:--enable-rbd)
 
-	# In "passthrough" security mode
-	# (-fsdev "...,security_model=passthrough,..."), qemu uses a helper
-	# application called virtfs-proxy-helper(1) to make certain 9p
-	# operations safer.
-	qemu_options+=(functionality:--enable-virtfs)
-	qemu_options+=(functionality:--enable-attr)
+	# WIth Kata we have virtio-fs and with CoCo we set shared_fs=nono
+	# there is no need for virtio-9p anymore
```

Comment:
@zvonkok - does this effectively remove 9p support? On the AC call the impression I got was that it was just pmem support that was blocking this PR, so we could do the 9p removal in a less rushed fashion. Did I misunderstand?

Reply:
Steve, you raised a very good point here.

I think 9p support from the runtime / guest kernel should be dropped together with this change, most likely on its own PR.

Reply:
@zvonkok - I'm not sure why this has been resolved, as we agreed to keep 9p until post-3.12?

Reply:
Ah shoot. I though I removed that.